src/Ajax.php

```diff
@@ -344,15 +344,33 @@ public static function createTabs(
 
             foreach ($tabs as $tab_key => $val) {
                 $target = str_replace('\\', '_', $val['id']);
-                $href = $val['url'] . (isset($val['params']) ? '?' . $val['params'] : '');
+                $tab_content_url = $val['url'] . (isset($val['params']) ? '?' . $val['params'] : '');
                 $selected = $active_id == $target ? 'selected' : '';
                 $title = $val['title'];
                 $title_clean = strip_tags($title);
+
+                // Compute direct link that user can reach in a new tab using
+                // middle mouse click.
+                // TODO: ctrl+click should have the same behavior but it seems
+                // to be caught by the tabs events handler and does not trigger
+                // a new browser tab.
```

Comment:
Clicks default behaviour are prevented by the bootstrap tabs logic. The only way to handle this is to remove the `data-bs-toggle='tab'` attribute and add a click handler that either let the default behaviour continue when `e.ctrlKey === true`, either trigger the tab display manually.

Reply:
Yeah, I am not sure it is worth dealing with right now as it doesn't break anything.